src/pages/gd/Swap/index.tsx

```diff
@@ -1,13 +1,13 @@
 import React, { memo } from 'react'
-import { UbeSwap } from './SwapCelo'
+import { UniSwap } from './SwapCelo'
```

Comment:
dont remove ubeswap file, create a new file for uniswap

Reply:
done